src/components/errorContent/Error.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Typography } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import theme from '../../styles/createTheme'
+
+const Error400 = () => {
```

Comment:
Plz change the names of component here (and in error 500), they are 400

Reply:
Done.